src/libs/actions/Search.ts

```diff
@@ -83,4 +86,23 @@ function deleteMoneyRequestOnSearch(hash: number, transactionIDList: string[]) {
     API.write(WRITE_COMMANDS.DELETE_MONEY_REQUEST_ON_SEARCH, {hash, transactionIDList}, {optimisticData, finallyData});
 }
 
-export {search, createTransactionThread, deleteMoneyRequestOnSearch, holdMoneyRequestOnSearch, unholdMoneyRequestOnSearch};
+type Params = Record<string, string | string[]>;
+
+function exportSearchItemsToCSV(query: string, reportIDList: string[] | undefined, transactionIDList: string[], policyIDs: string[]) {
```

Comment:
we will very soon remove query, do you how this will work then? Could you please ping here whoever is responsible for backend for CSV? 
in Search v2.1 we're moving to a parser and AST object for search

Reply:
cc @rlinoz

Reply:
This will need to keep parity with how search works, so we will need to update it to use the same AST object when it is available. I am not sure if this is part of Search v2.1 as well, but I think it should be since export is tied to search. 

cc: @luacmartins @trjExpensify

Reply:
Cool, sounds good. Glad it's on everyone's radar for when that is!

Reply:
We're moving to the jsonQuery AST in Search v2.1 and have already implemented parts of the changes, currently working on the final switch to the new syntax, so we'll need to update this code to work with the syntax.